osmaxx/excerptexport/management/commands/purge_old_result_files.py

```diff
@@ -54,10 +54,13 @@ def _remove_old_files(self):
                 export__output_file__file__isnull=False,
```

Comment:
Didn't notice the source filename, but now that you mention it, yes, that too.
![Change all the names](https://cdn.meme.am/instances/500x/69301335.jpg)